frontend/src/app/modules/availability/team/team.component.ts

```diff
@@ -11,15 +14,37 @@ import { SpinnerService } from '@core/services/spinner.service';
 export class TeamComponent implements OnInit {
     @Input() public teamSlot: IAvailabilitySlotsGroupByTeamsDto;
 
+    @Input() public currentUser: IUser;
+
     public slots: Array<IAvailabilitySlot>;
 
+    private unsubscribe$ = new Subject<void>();
+
     public teamName: string;
 
     // eslint-disable-next-line no-empty-function
-    constructor(public spinnerService: SpinnerService) {}
+    constructor(public spinnerService: SpinnerService, private availabilitySlotService: AvailabilitySlotService) {}
 
     ngOnInit(): void {
         this.slots = this.teamSlot.availabilitySlots;
         this.teamName = this.teamSlot.name;
     }
+
+    public updateTeamsSlots() {
+        this.availabilitySlotService
+            .getUserPersonalAndTeamSlots(this.currentUser.id)
+            .pipe(takeUntil(this.unsubscribe$))
+            .subscribe((resp) => {
+                const [teamSlots] = resp.teamSlots.filter((el) => el.name === this.teamName);
+
+                this.slots = teamSlots.availabilitySlots;
+                this.spinnerService.hide();
+            });
```

Comment:
This is not a good idea to reload all data to update 1 team
We have two ways how we can fix it:
1. Load and update all data in the user and teams page (like this idea more)
2. Add new endpoint to load specific team slots

Reply:
if I create 2 new endpoints (only for user's slots and only for team's slots), will it be good?

Reply:
You can, but I believe it will be easier to just reload all data

Reply:
I made it. Now all data is reloaded together